test/e2e/pull_test.go

```diff
@@ -84,6 +84,36 @@ var _ = Describe("Podman pull", func() {
 		Expect(session).Should(Exit(0))
 	})
 
+	It("podman pull and run on split imagestore", func() {
+		tmpDir, err := os.MkdirTemp("", "")
+		Expect(err).ToNot(HaveOccurred())
+
+		// First clear all images
+		session := podmanTest.Podman([]string{"rmi", "--all"})
+		session.WaitWithDefaultTimeout()
+		Expect(session).Should(Exit(0))
+
+		session = podmanTest.Podman([]string{"pull", "--imagestore", tmpDir, "quay.io/buildah/stable:latest"})
```

Comment:
`quay.io/buildah/stable:latest` is a gigantic image to pull.  We can avoid pulling entirely if we create an oci-archive of a local image and then do a `podman pull oci-archive:...`

Reply:
Switched to busybox.